markdown.go

```diff
@@ -380,6 +392,8 @@ func MarkdownOptions(input []byte, renderer Renderer, opts Options) []byte {
 		p.notes = make([]*reference, 0)
 	}
 
+	p.sanitizedAnchorNameOverride = opts.SanitizedAnchorNameOverride
```

Comment:
This would be where you'd check if `opts.SanitizedAnchorNameOverride` is `nil` and use an internal default value if so:

```Go
if opts.SanitizedAnchorNameOverride != nil {
    p.sanitizedAnchorName = opts.SanitizedAnchorNameOverride
} else {
    p.sanitizedAnchorName = sanitized_anchor_name.Create
}
```